transport/transport-backend-cct/src/main/java/com/google/android/datatransport/cct/CCTDestination.java

```diff
@@ -0,0 +1,48 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.android.datatransport.cct;
+
+import androidx.annotation.Nullable;
+import com.google.android.datatransport.runtime.Destination;
+
+public class CCTDestination implements Destination {
+  static final String DESTINATION_NAME = "cct";
+  private static volatile CCTDestination instance;
+
+  private CCTDestination() {}
+
+  public static CCTDestination getInstance() {
+    if (instance == null) {
+      synchronized (CCTDestination.class) {
+        if (instance == null) {
+          instance = new CCTDestination();
+        }
+      }
+    }
+    return instance;
+  }
+
+  @Nullable
+  @Override
+  public String getName() {
```

Comment:
Hm, not sure why this is `@Nullable`, should it be changed to `@NonNull` here and in the interface?

Reply:
Can't remember why. Resorting to Nonnull